src/ApprenticePortal/SFA.DAS.ApprenticePortal/InnerApi/ApprenticeAccounts/Requests/GetMyApprenticeshipRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+
+namespace SFA.DAS.ApprenticePortal.InnerApi.ApprenticeAccounts.Requests
+{
+    public class GetMyApprenticeshipRequest : IGetApiRequest
+    {
+        private readonly Guid _id;
+
+        public GetMyApprenticeshipRequest(Guid id)
+        {
+            _id = id;
```

Comment:
Hi Paul, worth making it variations of _apprenticeId/apprenticeId throughout rather than just Id? Or, we use _id/id and make sure all our new stuff is geared up in the same fashion